snowplow_tracker/emitters.py

```diff
@@ -323,7 +332,9 @@ def send_events(self, evts: PayloadDictList) -> None:
             if self._should_retry(status_code):
                 self._set_retry_delay()
                 self._retry_failed_events(failure_events)
+                self.event_store.cleanup(success_events, True)
```

Comment:
I think the `cleanup` function is also called from `_retry_failed_events` so no need to call here.